opennebula/defaults.yaml

```diff
@@ -99,10 +74,69 @@ kernel:
       controller_sshpubkeys: {}
 os_family:
   Debian:
+    oneadmin:
+      optional_groups:
+        - kvm
+        - libvirt
+        - ssl-cert
+    controller:
+      pkgs:
+        - opennebula
+        - opennebula-tools
+        - mysql-client
     compute_node:
       pkgs:
         - opennebula-node
+    oneflow:
+      pkgs:
+        - opennebula-flow
+        - ruby-treetop
+        - ruby-polyglot
+    onegate:
+      pkgs:
+        - opennebula-gate
+    sunstone:
+      pkgs:
+        - opennebula-sunstone
+        - python-numpy
   RedHat:
+    controller:
+      pkgs:
+        - opennebula-server
+        - mariadb
```

Comment:
Please comment this line out. I think it's better to install no DB libs/ driver by default.